oidc-ui/src/components/LoginQRCode.js

```diff
@@ -25,6 +28,7 @@ export default function LoginQRCode({
   const [status, setStatus] = useState({ state: states.LOADED, msg: "" });
   const [error, setError] = useState(null);
   const [qrCodeTimeOut, setQrCodeTimeout] = useState();
+  const [isMobileDevice, setIsMobileDevice] = useState(false);
```

Comment:
functionality around showing a button on mobile device, can be removed from here and raised in a different PR

Reply:
remove this particular functionality in this [f23c62b](https://github.com/mosip/esignet/pull/367/commits/f23c62bca823f172be40fc5ad7a41974c9ebf45b) and will create a separate PR